c7n/resources/vpc.py

```diff
@@ -191,6 +191,34 @@ def get_related_ids(self, resources):
         return vpc_group_ids
 
 
+@Vpc.filter_registry.register('vpc-attributes')
+class AttributesFilter(Filter):
+    """Filters VPCs based on their DNS attributes"""
+    schema = type_schema(
+        'vpc-attributes',
+        EnableDnsHostnames={'type': 'boolean'},
```

Comment:
We be generally gone for lower case hyphenated when not letting the user specify arbitrary keys

Reply:
gotcha. I had originally used lower case for the parameters but noted a few instances where upper case was used.